examples/java-export/.bazelversion

```diff
@@ -1 +1 @@
-7.1.0
+7.4.1
```

Comment:
7.5.0 is the latest stable 7.x release

Reply:
Ok, I'll update all bazel versions from 7.4.1 to 7.5.0